funsor/distributions.py

```diff
@@ -63,7 +66,7 @@ class Distribution(Funsor, metaclass=DistributionMeta):
         funsors or objects that can be coerced to funsors via
         :func:`~funsor.terms.to_funsor` . See derived classes for details.
     """
-    dist_class = "defined by derived classes"
+    dist_class = dist.Distribution
```

Comment:
Is this value actually used, or are you merely tidying up to be well-typed?

Reply:
Just tidying up to be well-typed.